modules/internal/BytesStringCommon.chpl

```diff
@@ -183,6 +178,59 @@ module BytesStringCommon {
     return ret;
   }
 
+   /*
+   This function decodeHelp is used to create a wrapper for 
+   qio_decode_char_buf* and qio_decode_char_buf_esc and return 
+   the value of syserr , cp and nBytes.
+   :arg buff: Buffer to decode 
+   :arg buffLen: Size of buffer
+   :arg offset: Starting index of read buffer,
+   :arg allowEsc: Choice between "qio_decode_char_buf" 
+                  and "qio_decode_char_buf_esc" for escape statement
```

Comment:
let's change "for escape statement" to "that allows escaped sequences in the string"

Make sure things fit in 80 columns after that change.

Reply:
Yep, made sure that these are less than 80 everywhere.